utils/paths.go

```diff
@@ -56,3 +57,11 @@ func GetCurationMavenCacheFolder() (projectDir string, err error) {
 	projectDir = filepath.Join(curationFolder, "maven", hex.EncodeToString(hasher.Sum(nil)))
 	return
 }
+
+func GetCurationPipCacheFolder() (string, error) {
```

Comment:
why Pip cache is a single folder and different from the one at maven where it is splited to a folder per wd?

Reply:
Thx for asking, in pip there is an issue with caching local packages that comes from artifactory, we know that today when u use redirect to s3 (as most of the client does) pip doesn not hit cache, see more in https://github.com/pypa/pip/issues/11701.

So not like maven, where we want for each separated project to check if the cache is empty and prompt a dedicated message, in pip it will not be true even if your cache for each project already full with packages.

Reply:
add comment for the project folder special case